packages/code-du-travail-frontend/src/outils/indemniteLicenciement/PrevNextBar.js

```diff
@@ -3,7 +3,9 @@ import PropTypes from "prop-types";
 import styled from "styled-components";
 import { Button, theme } from "@cdt/ui";
 
-function PrevNextBar({ onPrev, disabled, nextVisible, previousVisible }) {
+function PrevNextBar({ currentStepIndex, disabled, onPrev, stepsLength }) {
```

Comment:
I don't see the point of changing that, It implies to use PrveNextBar only in a model of with index base navigation :/

Reply:
I prefer to keep the logic located only where it's used. During the reducer refactoring I realised that the functions nextVisible and previousVisible were only used un PrevNextBar so I transfered it there